oxd-common/src/main/java/org/xdi/oxd/common/ErrorResponseCode.java

```diff
@@ -29,6 +29,7 @@
     PROTECTION_ACCESS_TOKEN_INSUFFICIENT_SCOPE(403, "protection_access_token_insufficient_scope", "protection_access_token does not have `oxd` scope. Make sure a) scope exists on AS b) register_site is registered with 'oxd' scope c) get_client_token has 'oxd' scope in request"),
     INVALID_NONCE(400, "invalid_nonce", "Nonce value is not registered by oxd."),
     INVALID_STATE(400, "invalid_state", "State value is not registered by oxd."),
+    INVALID_REFRESH_TOKEN(401, "invalid_refresh_token", "refresh_token is invalid. The provided refresh is not valid or is not longer valid."),
```

Comment:
401 is Unauthorized http status code. Please return 400.

Reply:
I think we've to rename the code, but not the number. If we pass-in invalid refresh token, it returns 401, and not 400

Reply:
Error says `invalid_refresh_token`, if we caught invalid token then for me 400 (bad request) looks better then 401 (Unauthorized). What do you think ? Also in oxauth code I think in majority of cases we return 400 and in two cases 401 (not sure why). I still propose on oxd side stick to 400.